.travis.yml

```diff
@@ -9,10 +11,19 @@ env:
 cache:
   directories:
     - "node_modules"
+before_install:
+  - chmod ugo+x ./test/install_chromedriver_travis.sh
```

Comment:
I think this can only wrapped into one script, maybe called `test_environment_setup.sh`?

Reply:
good idea